AirCasting/AirCastingApp.swift

```diff
@@ -14,6 +14,7 @@ struct AirCastingApp: App {
     @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
     
     @Environment(\.scenePhase) var scenePhase
+    @State var shoulProtect = false
```

Comment:
I suppose this is a typo mistake that should be changed in a few places. Anyway, everything is okay. 👍